packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -14,7 +14,7 @@ class RabbitEventMessageType(str, Enum):
 
 
 class RabbitMessageBase(BaseModel):
-    channel_name: str
+    channel_name: str = Field(..., const=True)
```

Comment:
From https://docs.pydantic.dev/usage/schema/#field-customization this argument
```
``const``: this argument must be the same as the field's default value if present.
```

What is the purpose? Here there is no default? Did you mean ``Config.frozen=True`` option in https://docs.pydantic.dev/usage/model_config/#options??

Reply:
it's a const value that can only have a specific value.